guides/policy/dealing_code_policy.md

```diff
@@ -0,0 +1,221 @@
+   
+   3.9. If you act as the trustee of a trust, you should speak to the Company Secretary, or designated director, about your obligations in respect of any Dealing in Company Securities carried out by the trustee(s) of that trust.
+   
+   3.10. You should seek further guidance from the Company Secretary, or designated director, before transacting in:
+   
+    >(A) units or shares in a collective investment undertaking (e.g. a UCITS or an Alternative Investment Fund) which holds, or might hold, Company Securities; or
+    >
+    >(D) financial instruments which provide exposure to a portfolio of assets which has, or may have, an exposure to Company Securities.
+
+   This is the case even if you do not intend to transact in Company Securities by making the relevant investment.
+   
+4. **Further guidance**
+
+    **If you are uncertain as to whether or not a particular transaction requires clearance, you must obtain guidance from the Company Secretary before carrying out that transaction.**
+
+## Part B – Additional provisions for PDMRs
+
+5. **Circumstances for refusal**
+
+    You will not ordinarily be given clearance to Deal in Company Securities during any period when there exists any matter which constitutes Inside Information or during a Closed Period. The Company may also consider it appropriate to withhold clearance when there is sensitive information relating to the Company (e.g. the Company is in the early stages of a significant transaction but the existence of such transaction does not yet constitute Inside Information). 
+
+6. **Notification of transactions**
+
+   6.1. You must notify the Company and the FCA in writing of every Notifiable Transaction in Company Securities conducted for your account as follows:
+    >(A) Notifications to the Company must be made using the template in Schedule 3 and sent to the Company Secretary, or designated director, by you or as soon as practicable and in any event within one business day of the transaction date. You should ensure that your investment managers (whether discretionary or not) notify you of any Notifiable Transactions conducted on your behalf promptly so as to allow you to notify the Company within this time frame.
+    >
+    >(E) Notifications to the FCA must be made within two working days of the transaction date electronically using the PDMR Notification Form on the FCA’s website <https://www.the-fca.org.uk/markets/ukla/regulatory-disclosures/mar-implementation>. If you would like, the Company Secretary can assist you with this notification, provided that you ask him to do so within one business day the transaction date.
```

Comment:
Can we use more inclusive language? Other instances of him/his/her/she etc further below. 

Perhaps prefer not to use a pronoun at all where possible e.g.
If you would like, ..., provided that it is asked for within one working day of the transaction date.

Reply:
Thanks Reuben, after consult with Debbie, the policy is updated to use more inclusive language.